batavia/utils.js

```diff
@@ -684,3 +684,28 @@ batavia.make_callable = function(func) {
     fn.__python__ = true;
     return fn;
 };
+
+/*************************************************************************
+ * utils
+ *************************************************************************/
+
+batavia.utils = batavia.utils || {};
+batavia.utils.isType = function isType(type, obj) {
+    var clas = Object.prototype.toString.call(obj).slice(8, -1);
+    return obj !== undefined && obj !== null && clas === type;
+};
```

Comment:
I took these out for two reasons - firstly, they're not needed, and secondly, they're based on a string comparison, so they're not reliable anyway.

As far as I can make out, you can do reliable type checks for everything we're likely to need with `instanceof` - if there's a case where this doesn't work, let me know and I can dig into this a bit deeper.